### DISCUSSION TITLE
혹시 예전처럼 1000ms이하로 여러개를띄워놓을수있는  시스템을 만들 계획이있으신가요???

### DISCUSSION THREAD

**Discussion by dudma123** [changdoc/naver-vaccine-macro#48] [Q&A]
몇일전까지 너무너무 잘쓰고있었는데 
확실히 한개만띄워놓으니까 
확률이 크게줄어들긴하네요..ㅠㅠㅠㅠ

혹시 계획이있으신가요??

**Comment by changdoc** [comment]
네이버 서버에서 계정 단위로 짧은 시간안에 예약 신청 페이지에 접근하게 되면 차단하고 있는 상황입니다.
서버측에서 제한을 둔 상황이라 구조적으로 이전의 형태로 동작하도록 불가할 것 같습니다.

그래서,
잔여 백신 맵 (https://m.place.naver.com/rest/vaccine?vaccineFilter=used) 상에서 새로고침 하면서 잔여 백신이 발생했을경우 해당 병원의 신청 페이지로 자동으로 이동하는 형태를 고민 중에 있긴합니다.

https://github.com/changdoc/naver-vaccine-macro/discussions/38#discussioncomment-1030829 
코멘트에 hw0603 님도 같은 형태로 제작을 진행하셨다고 하시네요.
